src/app/space/wizard/components/forge-app-generator/forge-app-generator.component.ts

```diff
@@ -46,75 +52,58 @@ export class ForgeAppGeneratorComponent implements OnInit, OnDestroy, OnChanges
   ngOnInit() {
     this.log(`ngOnInit ...`);
     this.appGenerator.commandName = this.forgeCommandName;
-    this.appGenerator.workflow = this.workflow;
+    // init Forge flow
+    this.appGenerator.begin();
+    ///// Wizard
+    this.wizardConfig = {
+      title: this.title,
+      hidePreviousButton: true, // TODO wizard: wait for merged and released https://github.com/patternfly/patternfly-ng/pull/112
+      hideIndicators: true, // TODO wizard: remove custom AppGeneratorStepViewComponent to reuse patterfly wizard step
+    } as WizardConfig;
   }
 
-  ngOnDestroy() {
-    this.log(`ngOnDestroy ...`);
-  }
-
-  /** handle all changes to @Input properties */
-  ngOnChanges(changes: SimpleChanges) {
-    for ( let propName in changes ) {
-      if ( changes.hasOwnProperty(propName) ) {
-        this.log(`ngOnChanges ... ${propName}`);
-        switch ( propName.toLowerCase() ) {
-          case 'workflow': {
-            let change: INotifyPropertyChanged<IWorkflow> = <any>changes[ propName ];
-            this.onWorkflowPropertyChanged(change);
-            break;
-          }
-          default : {
-            break;
-          }
-        }
-      }
-    }
-  }
+  stepChanged($event: WizardEvent): void {
```

Comment:
Shouldn't this be removed, since it is empty? If it is a next step or a ToDo item, a story should be creates in place of commented code.